Removing NGA as a department

bds002 opened the pull request:
NGA is not being contracted to return to MAGFest 2017
Removed them as a department and as a Schedule listing.

bds002 committed:
Removing NGA as a department

NGA is not being contracted to return to MAGFest 2017
Removed them as a department and as a Schedule listing.

bds002 committed:
Updates name of Event lead Tournaments in Gameroom

jcooter reviewed (approved):
+1

kitsuta commented:
As always, this will not actually remove anything from the database, where department and event rooms are stored.

kitsuta committed:
Use old var name so schedule will update correctly

Changing variable names for event locations simply adds new rooms - the room with the old variable is not removed unless we manually edit the DB. It's much easier to simply change the string associated with the old variable.
